Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.h

```diff
@@ -67,8 +73,11 @@ namespace AzToolsFramework
             Instance();
             explicit Instance(AZStd::unique_ptr<AZ::Entity> containerEntity);
             explicit Instance(InstanceOptionalReference parent);
+            explicit Instance(InstanceOptionalReference parent, InstanceAlias alias);
```

Comment:
Did some digging, after c++11 explicit with multiple params has meaning now that it can stop copy inits. I guess just keep it the way it is?
https://en.cppreference.com/w/cpp/language/explicit

Reply:
Why would you want to disable brace initialization?

Reply:
Good question! I'll have to think about that... 
Explicit maybe came along for the ride when copy+pasting the existing constructor. Looping in @srikappa-amzn (original author)

Reply:
No reason to disable brace initialization. explicit can be removed here.